packages/inter-protocol/src/provisionPoolKit.js

```diff
@@ -327,62 +341,96 @@ export const prepareProvisionPoolKit = (
             });
         },
         /**
-         * @param {object} [options]
-         * @param {MetricsNotification} [options.metrics]
+         * @param {ERef<Purse>} exchangePurse
+         * @param {ERef<Brand>} brand
          */
-        start({ metrics } = {}) {
+        watchCurrentAmount(exchangePurse, brand) {
           const {
             state: { brandToPSM, poolBrand },
             facets: { helper },
           } = this;
 
-          // Must match. poolBrand is from durable state and the param is from
-          // the contract, so it technically can change between incarnations.
-          // That would be a severe bug.
-          AmountMath.coerce(poolBrand, params.getPerAccountInitialAmount());
+          void observeNotifier(E(exchangePurse).getCurrentAmountNotifier(), {
```

Comment:
this is part of a commit titled `refactor:` but it's changing behavior. It's adding a new feature for upgrade durability so it's a `feat:`. Though it would have made review easier if the refactor changes were separated.

Reply:
fixed.   I'll try to make better separations.

Reply:
https://github.com/tummychow/git-absorb helps me. E.g. when you had the your review changes, if you ran `git add . && git absorb` it would make fixups on the previous. That way master doesn't have the iterations of code review.